### PR TITLE
Minor styling

### PR DESCRIPTION
**Pull request by Shades4355**
Styling for Index and Show pages:

Added background color
Added 'body' font tag
Added 'title' font tag
Added a couple font color tags

Changed Index page font size
--Updated Enzyme test to reflect <h3> to <h4> change

### CODE REVIEW

File: app/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -14,6 +14,7 @@ $lightgreen: #21b211;
```
Comment (jhayes00) [inline app/assets/stylesheets/main.scss:14]: looks good gang, can't wait to see it in action!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -6,26 +6,28 @@
...
+
+<body class='background-light-green'>
+
+<div class="custom-header">
+  <div class="grid-x grid-padding-x align-middle">
+    <div class="cell small-3">
+      <%= link_to 'Add A Game', "/" %>
+    </div>
+
+
```
Comment (safang99) [inline app/views/layouts/application.html.erb:18]: duplicate line here

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -6,26 +6,28 @@
...
+    </div>
+
+    <div class="cell small-3 text-right">
+      <%- if current_user -%>
+        <%= link_to 'Sign Out', destroy_user_session_path, method: :delete %>
+      <%- else -%>
+        <%= link_to 'Sign Up', new_user_registration_path %> |
+        <%= link_to 'Sign In', new_user_session_path %>
+      <%- end -%>
+
```
Comment (safang99) [inline app/views/layouts/application.html.erb:30]: remove empty line on 30

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (safang99) [reply app/views/layouts/application.html.erb:18]: just remove one of the empty lines

**Pull request merged**